docs/resources/Channel.md

```diff
@@ -896,27 +896,27 @@ You may create a message as a reply to another message. To do so, include a [`me
 
```

Comment:
There's references to the `embed` field in the warning and info sections for these endpoints which need to be updated